### PR TITLE
add config and docker compose templates

### PR DESCRIPTION
**Pull request by kashishshah**
## Context

Closes: #90 

- Add templates for client config and docker compose files.

---

## Reviewers

- @Lagrange-Labs/lsc-core-dev

### CODE REVIEW

File: templates/template_docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+services:
+  lagrange_attestation_node: # Modify if you want to change the service name
+    container_name: lagrange_attestation_node # Modify if you want to change the container name
+    image: lagrangelabs/lagrange-node:v0.4.0
+    restart: always
+    ports:
+      - "8080:8080"
+    environment:
+      - LAGRANGE_NODE_CLIENT_BLSKEYSTOREPATH=/app/config/keystore/bls.key
```
Comment (mastereng12) [inline templates/template_docker-compose.yml:11]: if it is required to fill those fields manually, you can remove this env, instead you can add `BLSKeystorePath = "/app/config/keystore/bls.key"` in config.toml

File: templates/template_client_config.toml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+[Client]
+GrpcURLs = "--- GrpcUrl of Chain ---"
+Chain = "--- Chain name ---"
+EthereumURL = "--- Ethereum RPC Endpoint of network (holesky/mainnet) ---"
+OperatorAddress = "--- EigenLayer Operator Address ---"
+CommitteeSCAddress = "0x57BAf26C77BBBa3D3A8Bd620c8d74B44Bfda8818"
+BLSKeystorePath = "--- BLS Keystore Path ---"
```
Comment (mastereng12) [inline templates/template_client_config.toml:7]: Please check the docker-compose comments, IMO, we can fill the following fields

**Review by mastereng12 — COMMENTED** [review]
overall, looks good direction. Do we need any documents here?

File: templates/template_docker-compose.yml (hunk 1 above)
Comment (kashishshah) [reply templates/template_docker-compose.yml:11]: > overall, looks good direction. Do we need any documents here?
Yes, I will add documentation in the next commit on this PR.

> if it is required to fill those fields manually, you can remove this env, instead you can add BLSKeystorePath = "/app/config/keystore/bls.key" in config.toml

My assumption is that the operators will use our CLI to either generate or import keystore. Considering that, I did not change this part as our CLI stores it under a certain path. Do you think I should still modify it as per your suggestion?

Comment (mastereng12) [reply templates/template_docker-compose.yml:11]: hmm, looks like we also need to update the cli logic, those key paths are duplicated in config and docker-compose.
We are binding the client.yml into the file of the docker container, don't think the local path is useful.

Comment (mastereng12) [reply templates/template_docker-compose.yml:11]: will make a quick PR for the reference!

Comment (mastereng12) [reply templates/template_docker-compose.yml:11]: I see, in the cli flow, the keystore  paths are referred from the client-config.toml, that's why we keep the above structure. but for the manual setup, you can change the structure like this

```yml
[Client]
GrpcURLs = "{{.ServerGrpcURL}}"
Chain = "{{.ChainName}}"
EthereumURL = "{{.EthereumRPCURL}}"
OperatorAddress = "{{.OperatorAddress}}"
CommitteeSCAddress = "{{.CommitteeSCAddress}}"
BLSKeystorePath = "/app/config/keystore/bls.key"
BLSKeystorePasswordPath = "/app/config/keystore/bls.pass"
SignerECDSAKeystorePath = "/app/config/keystore/signer.key"
SignerECDSAKeystorePasswordPath = "/app/config/keystore/signer.pass"
PullInterval = "1000ms"
BLSCurve = "{{.BLSCurve}}"

[RpcClient]
	{{ if eq .ChainName "optimism" "base" }}
	[RpcClient.Optimism]
	RPCURL = "{{.L2RPCEndpoint}}"
	L1RPCURL = "{{.L1RPCEndpoint}}"
	BeaconURL = "{{.BeaconURL}}"
	BatchInbox = "{{.BatchInbox}}"
	BatchSender = "{{.BatchSender}}"
	ConcurrentFetchers = "{{.ConcurrentFetchers}}"
	{{ else if eq .ChainName "arbitrum" }}
	[RpcClient.Arbitrum]
	RPCURL = "{{.L2RPCEndpoint}}"
	L1RPCURL = "{{.L1RPCEndpoint}}"
	BeaconURL = "{{.BeaconURL}}"
	BatchInbox = "{{.BatchInbox}}"
	ConcurrentFetchers = "{{.ConcurrentFetchers}}"
	{{ else if eq .ChainName "mock" }}
	[RpcClient.Mock]
	RPCURL = "{{.L2RPCEndpoint}}"
	{{ end }}
[Telemetry]
	ServiceName = "{{.MetricsServiceName}}"
	PrometheusRetentionTime = "{{.PrometheusRetentionTime}}"`
```

```yml
version: "3.7"

services:
  lagrange_client_{{.Network}}_{{.ChainName}}:
    container_name: lagrange_{{.Network}}_{{.ChainName}}_{{.BLSPubKeyPrefix}}
    image: {{.DockerImage}}
    restart: always
    ports:
      - "{{.PrometheusPort}}:8080"
    command:
      - "/bin/sh"
      - "-c"
      - "/app/lagrange-node run-client -c /app/config/client.toml"
    volumes:
      - {{.ConfigFilePath}}:/app/config/client.toml
      - {{.BLSKeystorePath}}:/app/config/keystore/bls.key
      - {{.BLSKeystorePasswordPath}}:/app/config/keystore/bls.pass
      - {{.SignerECDSAKeystorePath}}:/app/config/keystore/signer.key
      - {{.SignerECDSAKeystorePasswordPath}}:/app/config/keystore/signer.pass
      - lagrange_{{.Network}}_{{.ChainName}}_{{.BLSPubKeyPrefix}}:$HOME/.lagrange
    logging:
      driver: "json-file"
      options:
        max-size: "10m"
        max-file: "10"

volumes:
  lagrange_{{.Network}}_{{.ChainName}}_{{.BLSPubKeyPrefix}}:
```

File: templates/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    image: lagrangelabs/lagrange-node:{{.Version}}
+    restart: always
+    ports:
+      - "{{.PrometheusPort}}:8080"
+    command:
+      - "/bin/sh"
+      - "-c"
+      - "/app/lagrange-node run-client -c /app/config/client.toml"
+    volumes:
+      - {{.ConfigFilePath}}:/app/config/client.toml
```
Comment (mastereng12) [inline templates/docker-compose.yml:15]:
```suggestion
      - {{.ClientConfigFilePath}}:/app/config/client.toml
```

File: templates/client_config.toml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+	#{{ if eq .ChainName "optimism" "base" }}
+	[RpcClient.Optimism]
+	RPCURL = "{{.L2RPCEndpoint}}"
+	L1RPCURL = "{{.L1RPCEndpoint}}"
+	BeaconURL = "{{.BeaconURL}}"
+	BatchInbox = "{{.BatchInbox}}"
+	BatchSender = "{{.BatchSender}}"
+	ConcurrentFetchers = 8
+
+    #{{ else if eq .ChainName "arbitrum" }}
```
Comment (mastereng12) [inline templates/client_config.toml:25]:
```suggestion
	#{{ else if eq .ChainName "arbitrum" }}
```

**Review by mastereng12 — APPROVED** [review]
nit: some cleanup

**Pull request merged**